.circleci/test-deploy.yml

```diff
@@ -83,13 +83,29 @@ workflows:
           cypress-cache-key: cypress-cache{{ arch }}-{{ checksum "examples/yarn-install/package.json" }}
           post-install: "npx cypress install"
           package-manager: "yarn"
+      - cypress/run:
+          filters: *filters
+          name: Yarn Version Example
+          working-directory: examples/yarn-install
+          cypress-cache-key: cypress-cache{{ arch }}-{{ checksum "examples/yarn-install/package.json" }}
+          post-install: "./check-yarn-version.sh 1.22.5 && npx cypress install"
+          package-manager: "yarn"
+          package-manager-version: "1.22.5"
```

Comment:
Yarn `1.22.5` is not really a useful version anymore.
For Yarn Classic v1, it's probably better to use Yarn `1.22.22`

Reply:
I mean the usefulness isn't really relevant the main thing is it's not going to accidentally grab a pre-installed version